Singletons/IdentityProviders.gd

```diff
@@ -0,0 +1,26 @@
+extends Node
```

Comment:
Change to `extends Object`.

IdentityProvider should not extend Node because it doesn't use any Node functionality and there is also no `add_child()` called for it so it's currently an orphan node.

Reply:
If I change the type to `Object` I'm getting these errors when I try to create an instance of `IdentityProviders.ItchIndentityProvider.new(_http_request)` in runtime:
![image](https://github.com/Praytic/youtd2/assets/10060411/47a75961-65f4-4dad-803c-2c28d94838af)
![image](https://github.com/Praytic/youtd2/assets/10060411/0e3515dc-d531-493d-8a28-145be066a12d)

Reply:
I don't think that it's a problem that Singleton nodes are orphan. This is by design.

Reply:
Ok, I got confused. Singleton class `IdentityProviders` extending Node is okay.
But why is this file in singletons anyway? It's a normal class file, it should be somewhere in Scenes. No singleton functionality is used here.